src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
```

Comment:
If we are working with time, it is better to have the variable tell us in what units the value is stored.
For example, if the variable is storing seconds: `WORKING_TIME_IN_SECONDS`.